.github/workflows/python_package.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   test:
     name: ${{ matrix.task }} ${{ matrix.method }} (${{ matrix.os }}, Python ${{ matrix.python_version }})
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 60
+    timeout-minutes: 90
```

Comment:
Why was this necessary?

On latest `master`, it's taking only around 20 minutes for all `Python-package` GitHub Actions jobs to complete: https://github.com/microsoft/LightGBM/actions/runs/6097609206

Reply:
Doesn't seem to be, there's another issue that I mistook for a timeout. Will revert this change.